<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgLogin</class>
 <widget class="QDialog" name="dlgLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QLabel" name="appLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>87</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>xxx 客户端</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>62</y>
     <width>87</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLEdit">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>56</y>
     <width>187</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="logBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regBtn">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>206</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwdLabel_2">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>102</y>
     <width>87</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>口令：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLEdit_2">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>96</y>
     <width>187</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mLabel_3">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>142</y>
     <width>93</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>注册时随机数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mLEdit_3">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>138</y>
     <width>187</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
